app/modules/site_settings/helpers.py

```diff
@@ -265,6 +265,12 @@ def _validate_fields(cls, value, class_name):
                     f'{field_str} id {cf_cat_id}: displayType {display_type} not valid',
                 )
 
+            if display_type == 'multiselect' and not cf_def.get('multiple', False):
```

Comment:
Unless I'm misreading this, wouldn't we want to remove that `not`? If cf_def.get('multiple') returns true, I think this line would throw an exception.

Reply:
scratch that I misread it hah.